.gitignore

```diff
@@ -46,3 +46,5 @@
 
 # Embedded web-server pid file
 /.web-server-pid
+
+.idea
```

Comment:
Not sure about that, I don't think we should be IDE files in the .gitignore

Reply:
I approve you comment